src/app/route/CheckoutPage/CheckoutPage.style.scss

```diff
@@ -141,4 +141,12 @@
             margin-top: 0;
         }
     }
+
+    .Select-Container {
+            max-width: 40rem;
```

Comment:
Wrong nesting here.

Reply:
Moved max-width to select styling, all ul now has overflow: auto.